arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -1208,18 +1236,18 @@
       }
     },
     "CloudToDeviceProperties": {
-      "description": "The Iot Hub Cloud-To-Device messaging properties.",
+      "description": "The IoT hub cloud-to-device messaging properties.",
       "type": "object",
       "properties": {
         "maxDeliveryCount": {
-          "description": "The max delivery count for the device queue. Range : 1-100.",
+          "description": "The max delivery count for cloud-to-device messages in the device queue. See: https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-messaging#cloud-to-device-messages.",
```

Comment:
I purposefully avoided adding default values explicitly everywhere because we could change the defaults at any point and I would let documentation be the one place where all of the defaults and ranges are specified rather than having to deal with updating multiple artefacts whenever ranges or defaults change..so this was done on purpose